model/src/main/proto/topic.proto

```diff
@@ -139,6 +145,12 @@ message TopicList {
   repeated TopicSummary topic_summary = 2;
 }
 
+// Corresponds to the list of stories that have been completed.
```

Comment:
Is this in respect to a particular topic, or all topics? I think you should specify that in the documentation since it helps explain how to use this data.

Reply:
Updated: `// Corresponds to the list of stories that have been completed across all topics.`